components/map/layers/WMTSLayer.js

```diff
@@ -14,35 +14,52 @@ function getWMSURLs(urls) {
     return urls.map((url) => url.split("?")[0]);
 }
 
+function createWMTSSource(options) {
+    const urls = getWMSURLs(Array.isArray(options.url) ? options.url : [options.url]).map((url) => {
+        if (options.rev) {
+            return url + "?rev=" + options.rev
```

Comment:
I guess there is a slim chance that this may collide with a genuine query parameter of the service. I'd rather just pass the rev directly as a query key without value, which should still be a valid request, i.e.

                return url + "?" + options.rev